gapic/src/main/com/google/gapid/image/Histogram.java

```diff
@@ -87,13 +90,25 @@ public Range getInitialRange(double snapThreshold) {
 
   /**
    * @param percentile the percentile value ranging from 0 to 100.
+   * @param lastBin if true, for all the percentile bins from different channels, choose the last
+   * bin, otherwise choose the first bin.
    * @param high if true, return the upper limit on the percentile's bin, otherwise the lower limit.
    * @return the absolute pixel value at the specified percentile in the histogram.
    */
-  private double getPercentile(int percentile, boolean high) {
-    int bin = bins.getPercentileBin(percentile, channels);
-    return (bin < 0) ? mapper.limits.max :
-        getValueFromNormalizedX((bin + (high ? 1 : 0)) / (double)bins.count());
+  private double getPercentile(int percentile, boolean lastBin, boolean high) {
```

Comment:
Is there ever a reason `lastBin != high`? In all our usage we set them to the same value, so I think you can just use one parameter.

Reply:
Yes indeed, they are the same in all usage cases. Sounds good, just combined them and updated the annotations!